firefox-ios/Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -37,8 +37,7 @@ final class SearchSettingsTableViewController: ThemedTableViewController, Featur
             case .alternateEngines:
                 return .Settings.Search.AlternateSearchEnginesTitle
             case .preSearch:
-                // TODO: FXIOS-13644 - Add proper strings when finalized
-                return "Pre Search"
+                return ""
```

Comment:
So there's no strings at all here? Do we want to add a comment to say why this is normal 🤔 ?